src/Api/CreateOrderApi.php

```diff
@@ -83,6 +83,10 @@ public function create(string $token, PaymentInterface $payment, string $referen
                                 'currency_code' => $order->getCurrencyCode(),
                                 'value' => $payPalItemData['total_tax'],
                             ],
+                            'discount' => [
+                                'currency_code' => $order->getCurrencyCode(),
+                                'value' => abs($order->getOrderPromotionTotal()) / 100,
+                            ],
```

Comment:
I've checked these changes one more time. It seems we need such a fix in the `UpdateOrderApi` as well. When fixed, it will be good to be merged 🖖 🎉

Reply:
@LaskeJu you can fixed that please 🙂

Reply:
Thanks, changed it

Reply:
@Zales0123 it is ok for you ?